CONTRIBUTING.md

```diff
@@ -84,6 +84,53 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`.
 
+#### On a Windows machine:
+Install Node.js and npm. This page is helpful: http://blog.teamtreehouse.com/install-node-js-npm-windows <br>
+Install Ruby and the DevKit from here: http://rubyinstaller.org/downloads/
+```sh
+npm install --global gulp-cli
+```
+
+Then, to start, run the following command to install any new dependencies:
+
+```sh
+npm install
+```
+
+The following examples detail a few tasks you'll encounter as you use gulp:
+
+```sh
+gulp build
+```
+
+The task above is an alias for running `gulp sass javascript images fonts` and
+is the task to build all assets. Building the package will generate a `/dist`
+directory with the contents of the ZIP archive made available to download.
+Building just the package is useful if you'd like to create your own
+distribution bundle for frameworks that aren't supported via npm. The files in
+`/dist` contain no documentation and are compiled and bundled CSS, JavaScript,
+fonts, and images files. The command is aliased by `npm run prepublish`.
+
+```sh
+gulp website:build
+```
+
+The task above builds the entire Draft U.S. Web Design Standards website locally.
+It can be useful when debugging for build errors or generating a deployable
+version of the Standards website. This creates a `/_site` directory that
+contains the Jekyll-built site. This is the same build step that we use to
+deploy the website. The command is aliased by `npm run deploy'.
```

Comment:
is this "aliased by" construction a best practice, in this context? (I would like to avoid passive voice, if possible, but I understand completely if it's necessary in this context!)